Content.Server/UserInterface/UsingActivatableUIComponent.cs

```diff
@@ -0,0 +1,15 @@
+using Robust.Server.GameObjects;
+using Robust.Server.Player;
+using Robust.Shared.Reflection;
+using Robust.Shared.Serialization;
+
+
+namespace Content.Server.UserInterface
+{
+    [RegisterComponent]
+    public sealed class UsingActivatableUIComponent : Component
+    {
+        public ActivatableUIComponent UI = default!;
+    }
```

Comment:
What is this component used for?

Reply:
uhhh, one second, might be a merge thing